flocker/common/__init__.py

```diff
@@ -16,4 +17,5 @@
 from ._thread import auto_threaded
 from ._interface import interface_decorator
 from ._net import get_all_ips, ipaddress_from_string
-from ._retry import loop_until, poll_until, retry_failure
+from ._retry import loop_until, poll_until, retry_failure, \
+    retry_effect_with_timeout
```

Comment:
Use parens, rather than backslash. e.g. 

```python
from ._retry import (loop_until, poll_until, retry_failure,
    retry_effect_with_timeout)
```